.env.example

```
@@ -1,3 +1,15 @@
+# CookieKit analytics ID
 NEXT_PUBLIC_cookieKitId
+# Resend API key for email sending
 NEXT_PUBLIC_resend
-NEXT_PUBLIC_emailjs_dataEmail
+# Email address for contact form submissions
+NEXT_PUBLIC_dataEmail
+# Cloudflare Turnstile — bot protection for contact form
+NEXT_PUBLIC_TURNSTILE_SITE_KEY
+# Cloudflare Turnstile — server-side verification secret
+NEXT_PUBLIC_TURNSTILE_SECRET_KEY
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_TURNSTILE_SECRET_KEY key should be with a value or have an equal sign

(KeyWithoutValue)